File: CS/WindowsFormsApplication3/CustomTreeList.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraTreeList;
using DevExpress.XtraTreeList.Handler;
using DevExpress.XtraTreeList.Nodes;
using DevExpress.XtraTreeList.ViewInfo;
using DevExpress.Utils.Drawing;
using DevExpress.XtraTreeList.Painter;
using System.Reflection;

namespace WindowsFormsApplication3 {
    public class CustomTreeList : TreeList {
        public CustomTreeList() : base() { }

        protected override DevExpress.XtraTreeList.Handler.TreeListHandler CreateHandler() {
            return new CustomTreeListHandler(this);
        }
    }
}

File: VB/WindowsFormsApplication3/CustomTreeList.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraTreeList
Imports DevExpress.XtraTreeList.Handler
Imports DevExpress.XtraTreeList.Nodes
Imports DevExpress.XtraTreeList.ViewInfo
Imports DevExpress.Utils.Drawing
Imports DevExpress.XtraTreeList.Painter
Imports System.Reflection

Namespace WindowsFormsApplication3
	Public Class CustomTreeList
		Inherits TreeList
		Public Sub New()
			MyBase.New()
		End Sub

		Protected Overrides Function CreateHandler() As DevExpress.XtraTreeList.Handler.TreeListHandler
			Return New CustomTreeListHandler(Me)
		End Function
	End Class
End Namespace
